x-pack/plugins/spaces/public/share_saved_objects_to_space/components/share_to_space_flyout_internal.tsx

```diff
@@ -46,8 +46,17 @@ const LazyCopyToSpaceFlyout = lazy(() =>
 );
 
 const ALL_SPACES_TARGET = i18n.translate('xpack.spaces.shareToSpace.allSpacesTarget', {
-  defaultMessage: 'all',
+  defaultMessage: 'all spaces',
```

Comment:
uber-nit: If we delete this string from our translation files, then they will get re-translated to reflect your new default value here